provider-ui/.eslintrc.js

```diff
@@ -11,7 +11,8 @@ module.exports = {
     },
     "extends": [
         "eslint:recommended",
-        "plugin:react/recommended"
+        "plugin:react/recommended",
```

Comment:
Everything looks great! Can you please also update the eslint configuration of `/ui` folder?

Reply:
Sure.